english/transform/transform.md

```diff
@@ -5,4 +5,9 @@ tower over you. Clearly, you have shrunk.
 You feel... spongy. Squishy, even. You try to hold up your hands, but you don't have any! Looking down, you see the
 golden ring around your waist like a belt. The rest of you is white, soft, and cylindrical.
 
-You have turned into a marshmallow! The fear of chocolate and graham crackers fills you. 
+You have turned into a marshmallow! The fear of chocolate and graham crackers fills you.
+How will you escape this delicious nightmare?
+
+[Accept your fate and settle in to watch some netflix](../netflix/netflix.md)
+[Dance your blues away](../hammertime/hammertime.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.